app/__init__.py

```diff
@@ -40,3 +42,11 @@ def handleBaseException(error):
 # All resources that need to be routed should be routed here.
 api.add_resource(UserListResource, '/users', '/users/')
 api.add_resource(UserResource, '/users/<int:userId>', '/users/<int:userId>/')
+api.add_resource(todoMultiResource, '/todos', '/todos/')
+api.add_resource(todoResource, '/todos/<int:todoId>', 'todos/<int:todoId>/')
+api.add_resource(todoListMultiResource, '/todolist', '/todolist/')
+api.add_resource(todoListResource, '/todolist/<int:todoListId>',
+                 '/todolist/<int:todoListId>/')
```

Comment:
I typically see the resources of restful API be plural, but really consistency is all that matters.